node/src/utils/round_robin.rs

```diff
@@ -220,6 +227,25 @@ where
             .push_back(item)
             .await;
 
+        // NOTE: Count may be off by one b/c of the way locking works when elements are popped.
+        // It's fine for its purposes.
+        let total = self.queues.iter().map(|q| q.1.event_count()).sum::<usize>();
+        if let Some(recent_event_count_peak) = &self.recent_event_count_peak {
+            let recent_threshold = recent_event_count_peak.load(Ordering::SeqCst);
+            if total > recent_threshold {
```

Comment:
I wonder if it might be better to do something like
```rust
if total > (recent_threshold * 11) /10
```
so we only print if there's a 10% increase since the last time?

Reply:
Yes, good idea. Even with the basic checking I proposed we may still flood the logs. Update in https://github.com/casper-network/casper-node/pull/4295/commits/be55a4cbe914bf7b95db261e31bf855688dfc89a